packages/core/android/build.gradle

```diff
@@ -84,6 +84,10 @@ def getExtOrIntegerDefault(name) {
 android {
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
   buildToolsVersion getExtOrDefault('buildToolsVersion')
+  buildFeatures {
+      buildConfig = true
+  }
+  namespace = "com.datadog.reactnative"
```

Comment:
by having it here I guess you can now completely delete `AndroidManifest` file then.

Reply:
I will try, but I think doing so might break compatibility with RN < 0.71

Reply:
Ok I noticed the following behaviour:
- if I only remove the `package` field from the `AndroidManifest`, the build fails for RN < 0.70 (as already mentioned [here](https://github.com/react-native-community/discussions-and-proposals/issues/671#issuecomment-1597386954))
- if I remove the whole `AndroidManifest` file, then the build passes.

So does this mean we did not need this `AndroidManifest` file from the start?
Are there any functionality we might miss when removing it?

Reply:
you shouldn't need the manifest file, it is removed in SDK v2 of Android.

Reply:
Do you have documentation about the support of libraries without a manifest file? 
I could not find any docs mentioning that the manifest can be removed: https://developer.android.com/studio/projects/android-library

It seems to work fine with latest versions, but I want to be sure it won't break earlier versions of RN.